articles/api-auth/using-rules/_includes/_create-webtask.md

```diff
@@ -0,0 +1,18 @@
+Create the Webtask. You will need to set the following static metadata fields for the Webtask:
+
+* `wt-compiler = auth0-ext-compilers/${grant}`
+* `auth0-extension = runtime`
+* `auth0-extension-name = ${grant}`
+* `auth0-extension-secret = {random_secret}`
```

Comment:
The `${grant}` is actually a variable i'm passing in this doc in order to reuse its content. The `Using Rules with Client Credentials Grants` doc passes the value `client-credentials`, while `Using Rules with Resource Owner Password Grant` passes the value `password-exchange`. It looks like that:

![image](https://cloud.githubusercontent.com/assets/11715799/21933691/0f89a172-d9af-11e6-9d41-c8f9c0ffb2cf.png)